packages/client/components/SpotlightModal.tsx

```diff
@@ -192,6 +192,7 @@ const SpotlightModal = (props: Props) => {
               reflectionGroup={spotlightGroup}
               meeting={meeting}
               reflectionIdsToHide={reflectionIdsToHideRef.current}
+              expandedReflectionGroupPortalParentId='spotlight'
```

Comment:
Not sure if this was previously added and then removed while merging branches, but this prevents the Spotlight modal from closing when the expanded reflection stack closes

Reply:
yeah I've added that, we probably lost it along the way